CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 2.0.0
+
+## What's Changed
+
+### Breaking changes
+
+- Remove v1 algod API (`client.algod`) due to API end-of-life (2022-12-01). Instead, use v2 algod API (`client.v2.algod.algod`).
+- Remove `cost` field in `DryrunTxnResult` in favor of 2 fields: `budget-added` and `budget-consumed`. `cost` can be derived by `budget-consumed - budget-added`.
+- Remove logicsig templates (`logicTemplates`), `logic/langspec.json`, `logic.logic` depending on `langspec.json`.
+- Regenerate algod models so every constructor requires an object to be passed in. Previously, only constructors with more than 4 argument specified this.
+- Remove unused generated types: `CatchpointAbortResponse`, `CatchpointStartResponse`.
+- Remove following methods in favor of the methods with `WithSuggestedParams` suffix:
+  - `makePaymentTxn`, `makeKeyRegistrationTxn`, `makeAssetCreateTxn`, `makeAssetConfigTxn`, `makeAssetDestroyTxn`, `makeAssetFreezeTxn`, `makeAssetTransferTxn`.
+- Remove `makeLogicSig` in favor of either using `LogicSigAccount` (preferred) or directly invoking `LogicSig` constructor.
+- Remove `EncodedMultisigBlob` in favor of `EncodedSignedTransaction.
+
+### Enhancements
+
+- v2: Make breaking changes from v1 to v2.0.0 by @michaeldiamant in https://github.com/algorand/js-algorand-sdk/pull/717
```

Comment:
Optional - For this release, I think the already listed breaking changes account for the included changes.

```suggestion
```

Reply:
removed this bit.